olcut-core/src/test/java/com/oracle/labs/mlrg/olcut/command/StandardCommandTest.java

```diff
@@ -0,0 +1,108 @@
+package com.oracle.labs.mlrg.olcut.command;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import jdk.jfr.internal.LogLevel;
```

Comment:
Did you mean to import JFR here?

Reply:
Good catch - seems like the IDE shouldn't have done that without telling me. It's gone now (it must have come in why I typed LogLevel after meaning to type LogRecord).